lib/internal/worker.js

```diff
@@ -298,6 +305,16 @@ class Worker extends EventEmitter {
     }
   }
 
+  static #wrapUrlModule(url) {
```

Comment:
This feels wrong, we could probably handle this better but it doesn't have to block this PR

Reply:
Yeah, I definitely approve of a follow-up PR to handle this more appropriately